packages/styles/src/core/layout/map-wrapper/README.md

```diff
@@ -0,0 +1,187 @@
+## Map wrapper
+
+The map wrapper is used as the container for the map area and the bottom panel. The bottom panel is useful for long horizontal content.
+
+Inside the map area, you can add also floating panels, typically used for legends. These panels can be located in any of the four corners of the map.
+
+### Single map wrapper
+
+The most basic usage of the `as-map-wrapper` element is when you only need to show a map. No other panel. The map will cover the whole viewport.
+
+The map needs to be inside a `as-map` element like in this code snippet.
+
+```html
+noSource: true
+responsive: true
+---
+<iframe src="/examples/layouts/map-wrapper/single-map-wrapper.html" style="width: 100%; height: 100%;">
+```
+
+```code
+lang: html
+---
+<body class="as-app">
+  <main class="as-app-content">
+    <div class="as-map-wrapper">
+      <div id="as-map">
+        <div id="map"></div>
+      </div>
+    </div>
+  </main>
+</body>
+```
+
+
+### Map wrapper + Bottom panel
+
+To add a bottom panel you need to add an element with class `as-bottom-bar` as a child of `as-map-wrapper`.
+
+The bottom panel is hidden in mobile and visible in desktop size. The reason is to have the most visible map area possible in small screens. To make the panel visible you need to add the class `as-bottom-bar--visible` to the element that contains the `as-panels__bottom` panel.
```

Comment:
To make the panel visible `on small screen sizes` ...

Reply:
> you need to add the class as-bottom-bar--visible to the element that contains the as-panels__bottom panel.

This is not very clear to me

Reply:
Didn't see it on time. I'll do it in my next branch. Sorry.